package.json

```diff
@@ -20,5 +26,10 @@
   "bugs": {
     "url": "https://github.com/keepassxreboot/keepassxc-browser/issues"
   },
-  "homepage": "https://github.com/keepassxreboot/keepassxc-browser#readme"
+  "homepage": "https://github.com/keepassxreboot/keepassxc-browser#readme",
+  "browserslist": [
```

Comment:
Does this follow the minimum versions of Firefox specified in manifest.json?

Reply:
Decreased minimum supported Firefox version to 52